### ISSUE TITLE
Favicon not found running `npm run preview`

### ISSUE THREAD

**Issue by josecelano** [torrust/torrust-website-v1#176]
When you run the preview with `npm run preview` and you start navigating the page you will see some errors related to missing favicons.

```output
npm run preview

> torrust-website@1.0.0 preview
> vite preview


  ➜  Local:   http://localhost:4173/
  ➜  Network: use --host to expose
  ➜  press h to show help
NotFound [Error]: Not found: /favicon.ico
    at resolve2 (file:///home/josecelano/Documents/git/committer/me/github/torrust/torrust-website/.svelte-kit/output/server/index.js:2895:18)
    at resolve (file:///home/josecelano/Documents/git/committer/me/github/torrust/torrust-website/.svelte-kit/output/server/index.js:2722:34)
    at #options.hooks.handle (file:///home/josecelano/Documents/git/committer/me/github/torrust/torrust-website/.svelte-kit/output/server/index.js:2961:71)
    at respond (file:///home/josecelano/Documents/git/committer/me/github/torrust/torrust-website/.svelte-kit/output/server/index.js:2720:43)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  status: 404
}
NotFound [Error]: Not found: /blog/favicons/favicon.ico
    at resolve2 (file:///home/josecelano/Documents/git/committer/me/github/torrust/torrust-website/.svelte-kit/output/server/index.js:2895:18)
    at resolve (file:///home/josecelano/Documents/git/committer/me/github/torrust/torrust-website/.svelte-kit/output/server/index.js:2722:34)
    at #options.hooks.handle (file:///home/josecelano/Documents/git/committer/me/github/torrust/torrust-website/.svelte-kit/output/server/index.js:2961:71)
    at respond (file:///home/josecelano/Documents/git/committer/me/github/torrust/torrust-website/.svelte-kit/output/server/index.js:2720:43)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  status: 404
}
```

We also have that error in production:

![image](https://github.com/torrust/torrust-website/assets/58816/ef4df47f-d42b-4f9e-a303-aebbfcc78c5c)

**Comment by grmbyrn** [comment]
Hi @josecelano I think this issue has been resolved with [Add search bar, fix storybook, fix other minor errors #186](https://github.com/torrust/torrust-website/pull/186). I've tried to reproduce it but it's not showing up while navigating with `npm run preview` or in production.

**Comment by josecelano** [comment]
> Hi @josecelano I think this issue has been resolved with [Add search bar, fix storybook, fix other minor errors #186](https://github.com/torrust/torrust-website/pull/186). I've tried to reproduce it but it's not showing up while navigating with `npm run preview` or in production.

Hi @grmbyrn you are right.
